Intersect.Server/Database/Logging/Entities/ChatHistory.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Intersect.Enums;
+using System;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Intersect.Server.Database.Logging.Entities
+{
+    public class ChatHistory
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)] public Guid Id { get; private set; }
+
+        public Guid? UserId { get; set; }
+
+        public Guid? PlayerId { get; set; }
```

Comment:
Do not use `Guid?` for this. If there is no matching player, `PlayerId` should be set to `Guid.Empty`.